api/v1alpha1/osbuild_types.go

```diff
@@ -46,37 +46,28 @@ type TriggeredBy string
 // OSBuildStatus defines the observed state of OSBuild
 type OSBuildStatus struct {
 	// The conditions present the latest available observations of a build's current state
-	Conditions []OSBuildCondition `json:"conditions,omitempty"`
+	Conditions []Condition `json:"conditions,omitempty"`
 
 	// +optional
 	Output *string `json:"output,omitempty"`
 
 	// ComposeId presents compose id that was already started, for tracking a job of edge-container
 	// +optional
-	ContainerComposeId string `json:"containerComposeId,omitempty"`
+	ComposeId string `json:"containerComposeId,omitempty"`
 
-	// IsoComposeId presents compose id that was already started, for tracking a job of edge-installe
+	// AccessUrl presents the url of the image in S3 bucket
 	// +optional
-	IsoComposeId string `json:"isoComposeId,omitempty"`
-
-	// ContainerUrl presents the url of the edge-container image
-	// +optional
-	ContainerUrl string `json:"containerUrl,omitempty"`
-
-	// IsoUrl presents the url of the edge-installer image
-	// +optional
-	IsoUrl string `json:"isoUrl,omitempty"`
+	AccessUrl string `json:"accessUrl,omitempty"`
 
 	// +optional
 	// ComposerIso is the URL for the iso that composer build returns before
 	// packaing with the kickstart
 	ComposerIso string `json:"composer_iso,omitempty"`
 }
 
-type OSBuildCondition struct {
+type Condition struct {
```

Comment:
On one hand you generalized the name of the struct (same for `ConditionType`), but the description is still specific to OSBuild. Since we need to maintain a similar struct for the `OSBuildEnvConfig` and it seems like it is going to have the same structure, I think we need to generalize the descriptions as well

Reply:
Done